cmd/run.go

```diff
@@ -1,27 +1,47 @@
 package cmd
 
 import (
+	"errors"
+
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	"golang.org/x/sync/errgroup"
 )
 
 // runCmd represents the version command
 var runCmd = &cobra.Command{
-	Use:   "run <script>",
-	Short: "Executes a script",
-	Args:  cobra.ExactArgs(1),
-	Run: func(cmd *cobra.Command, args []string) {
-		_, _, script, _ := getTarget(args, true)
-		if script == nil {
-			log.Fatal("tree needs a package")
-		}
+	Use:   "run [scripts]",
+	Short: "Executes one or more scripts in parallel",
+	Long: `Executes one or more scripts in parallel
+All scripts will run to completion, regardless of whether or not the other scripts exit with errors.
+
+Should any of the scripts fail Leeway will exit with an exit code of 1 once all scripts are done executing.
+`,
+	Args: cobra.MinimumNArgs(1),
+	Run:  run,
+}
+
+func run(cmd *cobra.Command, args []string) {
```

Comment:
better keep this inline, because
- that's more consistent with the "Cobra style", and other commands
- `run` is fairly generic name